powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehavior.java

```diff
@@ -716,6 +729,62 @@ private Map<String, String> mapMerge(Map<String, String> m1, Map<String, String>
         return m3;
     }
 
+    @SneakyThrows(GenericServiceException.class)
+    private void generateAndSetOtpToOperationDetail(final OperationEntity operation, final OperationDetailResponse operationDetailResponse) {
+        operationDetailResponse.setTotp(generateTotp(operation, powerAuthServiceConfiguration.getProximityCheckOtpLength()));
+    }
+
+    private static String generateTotp(final OperationEntity operation, final int otpLength) throws GenericServiceException {
+        final String seed = operation.getTotpSeed();
+        if (seed == null) {
+            return null;
+        }
+
+        try {
+            return new String(Totp.generateTotpSha256(Base64.getDecoder().decode(seed), LocalDateTime.now(), otpLength));
```

Comment:
Can we split these calls into multiple lines?

Reply:
Sure.